angr/state_plugins/concrete.py

```diff
@@ -223,15 +230,20 @@ def _sync_simproc(self):
                 l.debug("Trying to re-hook SimProc %s", reloc.symbol.name)
                 # l.debug("reloc.rebased_addr: %#x " % reloc.rebased_addr)
 
-                #if self.state.project.arch.name is not 'ARMHF':
-                #    func_address = self.state.project.concrete_target.read_memory(reloc.rebased_addr, self.state.project.arch.bits / 8)
-                #    func_address = struct.unpack(self.state.project.arch.struct_fmt(), func_address)[0]
-                #else:
-
-                func_address = self.state.project.loader.main_object.plt[reloc.symbol.name]
+                if self.state.project.simos.name == 'Win32':
+                    func_address = self.state.project.concrete_target.read_memory(reloc.rebased_addr, self.state.project.arch.bits / 8)
```

Comment:
omg so much is going on here

- this is py3k, `/` is a float division operator
- we have `arch.bytes` which does this exactly, since
- `arch.byte_width` is a thing we try to respect generally